pom.xml

```diff
@@ -115,7 +115,9 @@
         <dependency>
             <groupId>com.github.lokka30.treasury</groupId>
             <artifactId>treasury-api</artifactId>
-            <version>1.0.0</version>
+            <!-- Force a specific commit since they created the 1.1.0 git tag days before -->
+            <!-- they actually released 1.1.0... so the existing JitPack build for 1.1.0 is incorrect -->
+            <version>937ec97a98</version> <!-- (1.1.0-937ec97-RELEASE) -->
```

Comment:
I just told JitPack to scrap the current version it has built. Does 1.1.0 work in pom now? :)

Reply:
No, because the `1.1.0` build is still from February 14: https://jitpack.io/com/github/lokka30/treasury/1.1.0/build.log

Reply:
JitPack is being weird. I can delete the old 1.1 builds, but they still seem to be used by our IDEs. And using this link instead, which just has a capital T for Treasury, yields the latest build for some reason.

https://jitpack.io/com/github/lokka30/Treasury/1.1.0/build.log